README.md

```diff
@@ -179,4 +179,4 @@ This project is an open Open Source project, please read `CONTRIBUTING.md`.
 
 We welcome pull requests, bug fixes and issue reports.
 
-Before proposing a large change, first please discuss your change by raising an issue.
```

Comment:
Whitespace?

Reply:
Did I load an unfortunate mistake ? My purpose would only concern license hyperlink's path

Reply:
Most certainly your editor added a newline to the last line wheras there was none before, no worry :)